.circleci/bin/apply.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/sh
```

Comment:
If y'all want to use the rackspace-toolbox container, it might reduce the pain of having to update these :)